### ISSUE TITLE
Could not load type 'AutoMapper.IMapperConfiguration' from assembly AutoMapper, Version=5.1.1.0

### ISSUE THREAD

**Issue by gurdev-r** [IdentityServer/IdentityServer3.Admin#45]
I recently upgraded my IdentityServer3 project to the latest build. In that same project I am using the package IdentityServer3.Admin.EntityFramework package to manage Clients and Scopes. Now when I run the project to access the admin panel, I get the following error.

```
{"Could not load type 'AutoMapper.IMapperConfiguration' from assembly 'AutoMapper, Version=5.1.1.0, Culture=neutral, PublicKeyToken=be96cd2c38ef1005'.":"AutoMapper.IMapperConfiguration"}

   at IdentityServer3.Admin.EntityFramework.IdentityAdminCoreManager`4..ctor(String connectionString, String schema, Boolean createIfNotExist)
   at Pablow.IdentityServer.Admin.IdentityAdminManagerService..ctor() in e:\Sonu\Dev\PablowWebApp\Src\Pablow\Pablow.IdentityServer\Admin\IdentityAdminManagerService.cs:line 12
   at lambda_method(Closure , Object[] )
   at Autofac.Core.Activators.Reflection.ConstructorParameterBinding.Instantiate() in c:\dev\git\IdentityServer3.Admin\source\Core\Configuration\Hosting\AutofacScope.cs:line 0
```

I guess it is due to the fact that latest IdentityServer3 package upgraded the AutoMapper to the latest version and Admin part is using the old version.

Can you please confirm or let me know how I can fix this?

**Comment by iBoonz** [comment]
The last few months I haven't got time to look at the Admin project, so it
is likely that AutoMapper 5.1.1.0 has breaking changes to the AutoMapper of
the EF project. I will try to fix this breaking change as soon as possible.
Maybe this week.

Currently if you want to fix this, you can pull the project from Git and
make the changes yourself.



2016-08-20 11:22 GMT+02:00 gurdev-r <notifications@github.com>:

**Comment by iBoonz** [comment]
Is fixed in 1.0.0-beta9
